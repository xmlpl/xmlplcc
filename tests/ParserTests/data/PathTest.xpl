node[] main() {
  node[] x;

  x/name0["pred1"==""]["pred2"==""]/name1["pred3"==""]["pred4"==""]/name2["pred5"==""]["pred6"==""];
}
