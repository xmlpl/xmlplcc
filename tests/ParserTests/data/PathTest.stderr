./data/PathTest.xpl:4:23: error: unexpected token: [
./data/PathTest.xpl:4:24: error: unexpected token: pred2
./data/PathTest.xpl:4:36: error: Fatal error during parse
